### PR TITLE
Fix copy block and duplicate of form fields

### LINKED ISSUE #336: Duplicating form fields removes original field
| Q | A
| --- | ---
| Bug? | yes
| New Feature? | no
| Bundle Version | 2.5
| Sulu Version | 2.5
| Browser Version | Browser name and version

#### Actual Behavior

If you edit a Sulu form in the admin and click on the duplicate button of a form field, it deletes the original version of the form field the moment you hit the save button. 

#### Expected Behavior

I expect that if i duplicate a form field both the original and the newly created form field keep existing after saving.

#### Steps to Reproduce

In a form click on the duplicate button of a form field. Click on the save button

### PR DESCRIPTION
**Pull request by alexander-schranz**
| Q | A
| --- | ---
| Bug fix? | yes
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | fixes #336
| Related issues/PRs | #issuenum
| License | MIT

#### What's in this PR?

Fix copy block for form manager

#### Why?

Currently copied field get removed as the same key is copied and key is used to find the field. If now multiple fields with the same key are copied we will ignore the second field and give it a new field.

### CODE REVIEW

File: Manager/FormManager.php [hunk 1]
```diff
@@ -357,6 +378,8 @@ protected function updateFields(array $data, Form $form, string $locale): void
             }
         }
 
+        exit;
+
```
Comment (alexander-schranz) [inline Manager/FormManager.php]:
```suggestion
```

**Comment by alexander-schranz** [comment]
@martinlagler Testcase added!

**Comment by popoplanter** [comment]
@alexander-schranz works in our project :)

File: Tests/Functional/Controller/FormControllerTest.php [hunk 2]
```diff
@@ -206,6 +206,79 @@ public function testPutFull(): void
...
+                        'type' => 'email',
+                        'title' => 'Title',
+                        'shortTitle' => 'Short Title',
+                        'placeholder' => 'Placeholder',
+                        'defaultValue' => 'Default Value',
+                        'width' => 'full',
+                        'required' => true,
+                    ],
+                    [
+                        'key' => 'email', // we are testing here if second email field is correctly created as email2
```
Comment (martinlagler) [inline Tests/Functional/Controller/FormControllerTest.php]: "third email field"

**Review by martinlagler — APPROVED** [review]

File: Tests/Functional/Controller/FormControllerTest.php (hunk 2 above)
Comment (alexander-schranz) [inline Tests/Functional/Controller/FormControllerTest.php]:
```suggestion
                        'key' => 'email', // we are testing here if third email field is correctly created as email2
```

**Pull request merged**